src/wp-includes/block-template-utils.php

```diff
@@ -184,6 +184,20 @@ function get_default_block_template_types() {
 	return apply_filters( 'default_template_types', $default_template_types );
 }
 
+/**
+ * Return a list of all overrideable default template type slugs.
+ *
+ * @see get_query_template
+ *
+ * @access private
+ * @since 5.9.0
+ *
+ * @return string[] List of all overrideable default template type slugs.
+ */
+function _get_template_type_slugs() {
+	return array_keys( get_default_block_template_types() );
+}
```

Comment:
Hmm, looks like the code in this function is only used in one place, i.e. in the `_add_template_loader_filters()` callback.

Do you foresee a future need to invoke this function elsewhere?

If no, then I'd suggest:

* Removing this function (one less function to maintain).
* Moving the code directly into `_add_template_loader_filters()`. The know-how and task here fits well into the processing work of `_add_template_loader_filters()`.

Reply:
Makes sense to me. The Gutenberg [counterpart](https://github.com/WordPress/gutenberg/blob/f0fc7c78b4ce4f50333202f0e48afceb604b9c53/lib/full-site-editing/default-template-types.php#L31-L40) has been around for a while and is still only used by `gutenberg_add_template_loader_filters`, so I think it's fine to inline. (And we can always add the function later if we find that we really need it -- easier than removing it.)

I'll take care of that.

Reply:
0a84ee9ea363683bbdedffaa93c4715886ac02a1

Reply:
Thank you @ockham!